### ISSUE TITLE
Dropdown: Accessibility test failing for an hidden-accessible input field

### ISSUE THREAD

**Issue by Bryelmo** [primefaces/primeng#12811]
### Describe the bug

Dropdown component fails a WCAG 2.1 AA standard test with **Axe Devtools** and **Accessibility insights for web** extensions on Google Chrome browser.

`<div class="p-hidden-accessible"><input type="text" readonly="" aria-haspopup="listbox" pautofocus="" role="combobox" class="p-element" aria-expanded="false"></div>`

AD and AIFW gives this feedback for this critical issue:

To solve this problem, you need to fix at least (1) of the following:

- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

### Environment

macOs Ventura 13.2.1 
Node 18.15.0
Npm 9.5.0

### Angular version

15.0.5

### PrimeNG version

15.2.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.15.0

### Browser(s)

Chrome 110.0.5481.177

### Steps to reproduce the behavior

Go to a page with PrimeNg Dropdown (es: https://primeng.org/dropdown) and run a test with Axe Devtools Chrome extension

### Expected behavior

No critical issue implementing one of the Axe Devtools suggestions
